### ISSUE TITLE
The effects of neuromorphic datasets

### ISSUE THREAD

**Issue by Lazy-LXH** [bic-L/Masked-Spiking-Transformer#6]
Dear bic-L,
Thank you for your work and open source code. I recently read the related paper "Masked Spiking Transformer" which provides the effects of neuromorphic datasets in your paper. I would appreciate it if you could provide the code to do the ANN-SNN conversion on neuromorphic datasets.

**Comment by bic-L** [comment]
Hi,
Did you mean the data loader part?

**Comment by bic-L** [comment]
We will update the code shortly, but adding a convolution layer for channel alignment as shown in Figure 4 of the main paper also works. The dataloader for the experiments can be found in another repository of ours: https://github.com/bic-L/burst-ann2snn/blob/main/dataset.py. Please let me know if you have any other questions!

Thanks

**Comment by Lazy-LXH** [comment]
I might have phrased my previous message incorrectly. I would like to replicate your work on implementing neuromorphic datasets through ANN-SNN transformations. Could you please provide the code for loading the data and the training process?

**Comment by bic-L** [comment]
Hi, 

We may update the repository in a couple of days, but are currently working to meet a deadline. The dataloader can be found in https://github.com/bic-L/burst-ann2snn/blob/main/dataset.py.  The training code is the same, while the main difference is in the model settings, which is shown in Figure 4 of the main paper. Hope this helps.
